cryptoconditions/types/zenroom.py

```diff
@@ -328,9 +293,9 @@ def validate(self, *, message):
         # We can put pulic keys either in the keys or the data of zenroom
         data.update(self.keys)
 
-        result = ZenroomSha256.run_zenroom(self.script,
-                                           {},
-                                           data)
+        result = zencode_exec(self.script,
+                              keys="{}",
+                              data=json.dumps(data))
         try:
```

Comment:
@albertolerda why do we need the metadata defined?
it's now mandatory for all integrations. why did you add this requirement?